src/missing_prices.py

```diff
@@ -103,7 +109,8 @@ def as_dune_repr(self, coin_id: str) -> dict[str, Any]:
 
 def load_tokens(dune: DuneClient) -> list[DuneRecord]:
```

Comment:
I think the return type should be list[TokenSchema] I am surprised mypy didn't catch this, but its probably because its not "--strict" (We should probably try to make it strict at some point).

Reply:
> I think the return type should be list[TokenSchema] I am surprised mypy didn't catch this, but its probably because its not "--strict" (We should probably try to make it strict at some point).

I think the actual return is a Dict because `.load` validates the input and puts it into a dict.